search/search_quality/assessment_tool/sample_view.hpp

```diff
@@ -0,0 +1,28 @@
+#pragma once
+
+#include "search/result.hpp"
+
+#include <QtWidgets/QWidget>
+
+class LanguagesList;
+class QLineEdit;
+class QListWidget;
+
+namespace search
+{
+struct Sample;
+}
+
+class SampleView : public QWidget
```

Comment:
What do you think about putting MainModel, SampleView and so on to a namespace? Is it generally possible in this case?